script/glam/test/test_glean_org_mozilla_fenix_glam_nightly

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# generate sql for checking into the repository and for testing the workflow
+
+set -e
+
+project=${PROJECT:-glam-fenix-dev}
+skip_generate=${SKIP_GENERATE:-false}
+# NOTE: there are three app_ids that we must look at for historical context. For
+# the purpose of this script, it is sufficient to look only at what is currently
+# "fenix nightly".
+app_ids=(
+    "org_mozilla_fenix"
+    "org_mozilla_fenix_nightly"
+    "org_mozilla_fennec_aurora"
+)
+logical_app_id="org_mozilla_fenix_glam_nightly"
+
+dir="$(dirname "$0")/.."
+sql_dir=$dir/../../sql/$project/glam_etl
+
+if [[ $skip_generate == false ]]; then
+    # generate views for each of the referenced products to avoid errors
+    for app_id in "${app_ids[@]}"; do
+        PRODUCT=$app_id STAGE=daily $dir/generate_glean_sql &
+    done
+    wait
+    for app_id in "${app_ids[@]}"; do
+        # keep all pings for org_mozilla_fenix, and only the baseline ping for
+        # the others
+        if [[ $app_id == "org_mozilla_fenix" ]]; then
+            continue;
+        fi
+        for path in "${sql_dir}/${app_id}__clients"*; do
+            if [[ $path == *metrics* ]]; then
```

Comment:
In case the next firefox is `org_mozilla_metrics` or something:
```suggestion
            if [[ $path == "${sql_dir}/${app_id}__clients"*metrics* ]]; then
```

Reply:
This is a little too strict.

Reply:
Actually, this works fine, I forgot the middle glob.